packages/use-contractkit/__tests__/connectors/walletconnect.test.ts

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
```

Comment:
What is this for?

Reply:
It disables eslint worrying about 

`@ts-ignore` and `@ts-expect-error`

I think we should disabled the rule altogether but for now i just move it above the line that needs it